clients/drcachesim/tracer/raw2trace_shared.h

```diff
@@ -73,6 +73,10 @@ namespace drmemtrace {
 #define WINDOW_SUBDIR_FIRST "window.0000"
 #define TRACE_SUBDIR "trace"
 #define TRACE_CHUNK_PREFIX "chunk."
+// The chunk name is "chunk.nnnnnnnn" with leading 0's for the number suffix.
+// Some zip readers end up alphabetizing the component files, so be sure to have enough
+// leading zeroes to keep them in numeric order even with large component counts.
```

Comment:
> so be sure to have enough leading zeroes to keep them in numeric order even with large component counts.

raw2trace_t can do this check perhaps. Or just an XXX comment for now.

Reply:
Added a check to raw2trace.  Updated the PR description with manual testing done for this check.